src/lib/actions/client.ts

```diff
@@ -133,3 +133,279 @@ export async function removeProjectMember(proj_id: string, user_id: string) {
 
 	return true;
 }
+
+export async function updateTask(task: ITasks) {
```

Comment:
remove function before merging to. main. thoi update task accessing from `tasktmp` table.